fft_simulation/analytical_cases.py

```diff
@@ -35,18 +68,33 @@ def analyticial_sol(self):
         r = np.sqrt(x**2 + y**2 + z**2)
 
         Bz_out = self.sus_diff/3 * (self.R/r)**3 * (3*z**2/r**2 - 1)
-        Bz_in = np.zeros(self.matrix)
-
+        Bz_out[mask] = 0
 
-        Bz_in = np.where(mask, Bz_in, 0)
-        Bz_out = np.where(~mask, Bz_out, 0)
+        Bz_in = np.zeros(self.matrix)
 
         Bz_analytical = Bz_out + Bz_in
```

Comment:
What is the point of Bz_in? It's only a matrix of zeros.

Reply:
will remove